modules/ROOT/pages/logging-in-mule.adoc

```diff
@@ -0,0 +1,287 @@
+= Configuring Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, studio, logger, logs, log, notifications, errors, debug
+
+Mule automatically logs multiple messages and specific elements in your application flows to help you debug and keep track of events. You can also include the xref:logger-component-reference.adoc[`<logger>` Element] anywhere in a flow and set it up to output any message you want.
```

Comment:
"Element" -> "element" We generally refer to this as the Logger component unless they are using the XML config for this.

Reply:
Fixed